types/options.go

```diff
@@ -424,3 +425,40 @@ func ReloadConfigurationFile(configFile string, storeOptions *StoreOptions) {
 func Options() StoreOptions {
 	return defaultStoreOptions
 }
+
+// SaveLocalStorageConfig overwrites the tomlConfig in storage.conf with the given conf
+func SaveLocalStorageConfig(conf TomlConfig) error {
+	configFile, err := DefaultConfigFile(unshare.IsRootless())
+	if err != nil {
+		return err
+	}
+	if _, err := os.Stat(configFile); err == nil {
+		err = os.Remove(configFile)
```

Comment:
One line 
if err = os.Remove(configFile); err {
...

Reply:
I think this will work, right?

```
if err = os.Remove(configFile); os.IsNotExist(err) {
	return err
}
```

Reply:
Yup